javascript/src/openapi/http/isomorphic-fetch.ts

```diff
@@ -24,7 +24,9 @@ export class IsomorphicFetchHttpLibrary implements HttpLibrary {
         }
       };
       await sleep(nextInterval);
-      return await this.sendWithRetry(request, --triesLeft, nextInterval * 2);
+      const headers = request.getHeaders();
+      headers['svix-retry-count'] = `${retryCount}`
```

Comment:
```suggestion
      headers['svix-retry-count'] = retryCount.toString()
```
Double check me please.